WalletWasabi.Fluent/Helpers/TransactionHelpers.cs

```diff
@@ -99,12 +99,9 @@ public static async Task<SmartTransaction> ParseTransactionAsync(string path, Ne
 		{
 			psbt = PSBT.Load(psbtBytes, network);
 		}
-		catch (FormatException ex)
-		{
-			throw new FormatException("An error occurred while loading the PSBT file.", ex);
```

Comment:
I would replace this by a `LogWarning()`.

Reply:
I will add  couple of `LogWarning`s similar to the comments.